### PR TITLE
add aumz work

### PR DESCRIPTION
**Pull request by alabahuy**
## Description

add aumz work support via, hotswap and soldered version

## QMK Pull Request

https://github.com/qmk/qmk_firmware/pull/23600

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have formatted the JSON file to have consistent formatting with the rest of the repository.
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/jaykeeb/aumz_work/aumsworkhotswap.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
+{
+  "name": "Aums Work Hotswap Version",
```
Comment (Xelus22) [inline v3/jaykeeb/aumz_work/aumsworkhotswap.json:2]: `Aums` or `Aumz`?

Probably dont need `Version`?

File: v3/jaykeeb/aumz_work/aumsworksoldered.json [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+  "productId": "0x0772",
+  "matrix": {"rows": 5, "cols": 15},
+  "layouts": {
+    "labels": [
+      "Split Backspace",
+      "ISO Enter",
+      "Split LShift",
+      ["Bottom Row", "7U", "6.25U"]
+    ],
+    "keymap": [
```
Comment (Xelus22) [inline v3/jaykeeb/aumz_work/aumsworksoldered.json:13]: image

Key is missing group

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/jaykeeb/aumz_work/aumsworkhotswap.json (hunk 1 above)
Comment (alabahuy) [reply v3/jaykeeb/aumz_work/aumsworkhotswap.json]: sorry, aumz is right. yes dont need version

File: v3/jaykeeb/aumz_work/aumsworksoldered.json (hunk 2 above)
Comment (alabahuy) [reply v3/jaykeeb/aumz_work/aumsworksoldered.json]: done

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**